src/main/scanImages.ts

```diff
@@ -102,33 +89,18 @@ export function scanImagesFactory(serviceContainer: LocalMain.ServiceContainerSe
 				}, 0
 			);
 
-			const compressedImagesOriginalSize = await Object.values(updatedImageData).reduce(
-				(acc, data) => {
-					if (data.compressedImageHash) {
-						return acc + data.originalSize;
-					} else {
-						return acc;
-					}
-				}, 0
-			);
-
-			const nextSiteImageData = {
+			const nextSiteImageData: SiteImageData = {
 				...siteData,
 				imageData: updatedImageData,
-				scanInProgress: false,
```

Comment:
With this value no longer being written to disk at the end of a scan, this means that the "Scan" button can get stuck in "Scanning...." state if you navigate away from the tab while a scan is in progress. 

You could look at rolling this in to the next ticket to persist state while away from tab, but also might be nice to just clean it up now.

Reply:
This will be automatically fixed with the LOC-2232 work (confirmed when I started in on this last week). I'm hesitant to fix it up here because the level of effort to do so with the existing setup is rather high and we will be including LOC-2232 with the next release.

I could also apply a partial patch of the LOC-2232 branch to this branch.

Thoughts?